src/macos-symbol-plane.ts

```diff
@@ -0,0 +1,96 @@
+/**
+ * Map of special symbols to the keys that would be pressed (while holding `Option`) to type them on MacOS on an
+ * English layout. Most of these are standardized across most language layouts, so this won't work 100% in every
+ * language but it should work most of the time.
+ */
+export const macosSymbolPlaneKeys: Record<string, string> = {
```

Comment:
super nitpick that the noun I'd expect would be "layer" instead of "plane" to describe this structure due to [keyboard hobbyist land](https://keebmaker.com/pages/layers), would be nice to use here for greppability

Reply:
Makes sense! I was looking for a good word to describe this and I agree that "layer" is more intuitive. It doesn't make me thing of airplanes 😄